File: 백준/Bronze/2446. 별 찍기 － 9/별 찍기 － 9.js
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().trim();

const N = +input;
const star = "*";
const _ = " ";

for (let i = N; i > 0; i--) {
  console.log(_.repeat(N - i) + star.repeat(2 * i - 1));
}
for (let i = 2; i <= N; i++) {
  console.log(_.repeat(N - i) + star.repeat(2 * i - 1));
}

File: 백준/Bronze/2446. 별 찍기 － 9/별 찍기 － 9.java
import java.io.*;

public class Main {

	public static void main(String[] args) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
		StringBuilder sb = new StringBuilder();

		int N = Integer.parseInt(br.readLine());

		for (int i = 0; i < N; i++) {
			String result = "";
			result += " ".repeat(i);
			result += "*".repeat(2*(N-i)-1);

			sb.append(result).append("\n");
		}

		for (int i = 1; i < N; i++) {
			String result = "";
			result += " ".repeat(N-1-i);
			result += "*".repeat(2*i+1);

			sb.append(result).append("\n");
		}

		bw.write(sb.toString().trim());
		bw.flush();
		bw.close();
		br.close();
	}
}